schemas/theia-extension.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Theia package.json Schema",
+  "type": "object",
+  "$ref": "https://json.schemastore.org/package.json",
+  "required": [
+    "keywords",
+    "theiaExtensions"
+  ],
+  "properties": {
+    "keywords": {
+      "description": "This helps people discover your package as it's listed in 'npm search'. Must include 'theia-extension'.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "contains": {
+        "const": "theia-extension"
+      },
+      "minItems": 1
```

Comment:
The error I get when not fulfilling this is `Array does not contain required item.` which might be confusing if you do not know the schema. Not sure if we really want to enforce this but I'm open to it.